docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -135,23 +117,23 @@ For more information see reference at the [Hardware specification](https://githu
 |STM32H743|STM32H743|
 |STM32H750|STM32H750|
 
-- find and install the firmware (4.0 or newer) for the Unified Target identified above. The firmware is available from the board type drop-down in configurator. Be aware that after flashing this firmware, the LEDs on your board will not be working - this is normal.
+- find and install the firmware (4.0 or newer) for the Target identified above. The firmware is available the releases page as assets (generic) for Betaflight, or you may need to build your own. Be aware that after flashing this firmware, the LEDs on your board will not be working - this is normal.
```

Comment:
expand the 'build your own' section to explain how to pass all the required defines on the command line to the build system, include examples of the edge cases that include quoted strings brackets and other things which will cause people issues when doing this due to command line interface differences.  give at least a `bash` example.